### PR TITLE
Add default value for RouteReconciliationPeriod in cloud controller manager

### PR DESCRIPTION
**Pull request by karataliu**
**What this PR does / why we need it**:
Add default sync period value config for RouteReconciliationPeriod. For now the default value is 0, which means zero cooldown time.

The value is taken from kube-controller-manager:
https://github.com/kubernetes/kubernetes/blob/b2b079b95a867b560fd327cba56e7d3878f43dc3/cmd/kube-controller-manager/app/options/options.go#L73

**Which issue this PR fixes**

**Special notes for your reviewer**:

**Release note**:

### CODE REVIEW

File: cmd/cloud-controller-manager/app/options/options.go [hunk 1]
```diff
@@ -46,18 +46,19 @@ type CloudControllerManagerServer struct {
...
+			MinResyncPeriod:           metav1.Duration{Duration: 12 * time.Hour},
+			NodeMonitorPeriod:         metav1.Duration{Duration: 5 * time.Second},
+			ClusterName:               "kubernetes",
+			ConfigureCloudRoutes:      true,
+			ContentType:               "application/vnd.kubernetes.protobuf",
+			KubeAPIQPS:                20.0,
+			KubeAPIBurst:              30,
+			LeaderElection:            leaderelectionconfig.DefaultLeaderElectionConfiguration(),
+			ControllerStartInterval:   metav1.Duration{Duration: 0 * time.Second},
+			RouteReconciliationPeriod: metav1.Duration{Duration: 10 * time.Second},
```
Comment (thockin) [inline cmd/cloud-controller-manager/app/options/options.go]: Please add a comment to this function that many values are copied from the original controller manager and should probably stay in syc, and add a reciprocal comment to the original location.

Comment (karataliu) [reply cmd/cloud-controller-manager/app/options/options.go]: Sounds good. Note added.

**Pull request merged**